code/drasil-website/lib/Drasil/Website/About.hs

```diff
@@ -6,15 +6,22 @@ import Language.Drasil
 
 -- * About Section
 
+-- | Reusable wiki page sentence.
+wikiSentenceComma :: Sentence
+wikiSentenceComma = S "wiki page,"
```

Comment:
Instead of defining this, you should just use the `sC` combinator to combine a `Sentence` with another `Sentence`, inserting a `Comma` in between.